spec/features/admin/admin_adds_items_spec.rb

```diff
@@ -31,5 +40,20 @@
     expect(page).to have_content "Horrorland II"
     expect(page).to have_content "Double scary"
     expect(page).to have_content "$59.95"
+    expect(page).to have_content "Monster"
+  end
+
+  scenario 'cannot add new item without a category' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.